src/emmodel/model.py

```diff
@@ -40,14 +40,16 @@ def __init__(self,
         ]
         self.models = []
         self.results = []
+        self.vcovs = []
 
     def run_models(self):
         for i in range(self.num_models):
             self.data[i].attach_df(self.df)
             self.models.append(self.model_variables[i].get_model(self.data[i]))
             self.results.append(scipy_optimize(self.models[i]))
+            self.vcovs.append(self.models[i].opt_vcov)
             pred = self.models[i].params[0].get_param(
-                self.results[i]["coefs"], self.data[i]
+                self.results[i], self.data[i]
```

Comment:
might be good to change `self.results` to `self.coefs` or directly use `self.models[i].opt_coefs` to avoid confusion.